### PR TITLE
Added index for lh_sample_uuid to the samples collection

### PR DESCRIPTION
**Pull request by andrewsparkes**

### CODE REVIEW

File: crawler/main.py [hunk 1]
```diff
@@ -72,6 +73,10 @@ def run(sftp: bool, keep_files: bool, add_to_dart: bool, settings_module: str =
                 logger.debug(f"Creating index '{FIELD_RESULT}' on '{samples_collection.full_name}'")
                 samples_collection.create_index(FIELD_RESULT)
 
+                # Index on lh_sample_uuid column to make it easier for queries joining on the samples from lighthouse
+                logger.debug(f"Creating index '{FIELD_LH_SAMPLE_UUID}' on '{samples_collection.full_name}'")
+                samples_collection.create_index(FIELD_LH_SAMPLE_UUID)
```
Comment (pjvv) [inline crawler/main.py:78]: Looks good, as long as it achieves the same as what @emr did manually?

Comment (andrewsparkes) [reply crawler/main.py:78]: I can see the index that @emr made on the samples collection in UAT. If this makes a second instance of it somehow we can drop his version.

Comment (emrojo) [reply crawler/main.py:78]: index was with sparse=true so it wouldnt' include documents that don't have this attribute.

Comment (andrewsparkes) [reply crawler/main.py:78]: ok will add, should it be both **sparse** and **unique** do you think?

**Comment by andrewsparkes** [comment]
Tested in UAT and worked (dropped index manually then ran crawler to recreate index). Took a couple minutes to create the index.

**Pull request merged**